VEDA_OS01/models.py

```diff
@@ -463,3 +480,30 @@ class Meta:
 
     def __unicode__(self):
         return u'{org} - {provider}'.format(org=self.org, provider=self.provider)
+
+
+class TranscriptProcessMetadata(TimeStampedModel):
+    """
+    Model to contain third party transcript process metadata.
+    """
+    video = models.ForeignKey(Video)
+    provider = models.CharField('Transcript provider', max_length=50, choices=TranscriptProvider.CHOICES)
+    process_id = models.CharField('Process id', max_length=255)
+    lang_code = models.CharField('Language code', max_length=3)
+    status = models.CharField(
+        'Transcript status',
+        max_length=50,
+        choices=TranscriptStatus.CHOICES,
+        default=TranscriptStatus.PENDING
+    )
+
+    class Meta:
+        unique_together = ('video', 'provider', 'lang_code')
```

Comment:
TODO: we need to remove this constraint, this will raise integrity in case of VEDA heal (i.e. transcription re-process for same lang)